src/main/kotlin/com/team4099/robot2022/commands/led/ChangeLedState.kt

```diff
@@ -0,0 +1,56 @@
+package com.team4099.robot2022.commands.led
+
+import com.team4099.robot2022.config.Constants
+import com.team4099.robot2022.subsystems.LED
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class ChangeLedState(val changestate: Constants.LED.LEDState) : CommandBase() {
+  init {
+    addRequirement(Intake)
+    addRequirement(Shooter)
+    addRequirement(LED)
+  }
+
+  override fun initialize() {
+    val isChanging = false
+
+    if (intakeTalon.supplyCurrent >= Constants.LED.CURRENT_THRESHOLD){
+      LED.ballCount = LED.ballCount + 1
+    }
+    else {
+      LED.intakeReady = true
+    }
```

Comment:
instead of having a ball count, just do if we have a ball. so make it a boolean value and it turns true when intake exceeds a certain current and turn false when shooter turns a certain current.